piker/clearing/_client.py

```diff
@@ -175,6 +175,7 @@ async def relay_order_cmds_from_sync_code(
 async def open_ems(
     fqsn: str,
     mode: str = 'live',
+    loglevel: str = 'error',
```

Comment:
Yeah i think ideally in the longer run we have a better "tagging" log level system such that we can flick on particular "layers" of the stack with different levels of "granularity" (which is more like the built-in python leveling system).

I think some of this is discussed in https://github.com/goodboy/tractor/issues/244 and